.github/PULL_REQUEST_CP_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+**This pull request is opened against a release branch.<br>
+To request a cherry pick of a commit, please fill in the form below.** (Questions with an asterisk are required.)<br>
+
+**To fill in the form, you can edit this PR description and type your answers after the 'My Answer' keywords. <br>
+A flutter domain expert will evaluate this cherry pick request shortly after ALL questions are answered.**
+
+* Changelog Description: Explain this cherry pick in one line that is accessible to most Flutter developers
```

Comment:
This is missing the section with the original issue

Reply:
Sure, added the original issue field.

Also added text box styling around the answers fields, it now looks like the PR description of https://github.com/flutter/flutter/pull/139594.

I looked into [colored blocks](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) but looks like they are for alerts and not regular text blocks.